library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt

```diff
@@ -30,11 +34,17 @@ data class TooltipViewState(
         }
     }
 
-    fun getTopArrowVisibility() =
-        if (arrowPosition == ArrowPosition.UP) View.VISIBLE else View.GONE
+    fun getTitleVisibility() = titleText.isNotEmpty()
 
-    fun getBottomArrowVisibility() =
-        if (arrowPosition == ArrowPosition.DOWN) View.VISIBLE else View.GONE
+    fun getDescriptionVisibility() = descriptionText.isNotEmpty()
+
+    fun getTopArrowResource() = if (arrowResource == Constants.DEFAULT_ARROW_RESOURCE) R.drawable.ic_arrow_up else arrowResource
```

Comment:
can you add unit test for new functions

Reply:
Unit test class for TooltipViewState created. You can check on the last commit